src/System.Private.CoreLib/shared/System/Threading/EventWaitHandle.cs

```diff
@@ -22,7 +22,7 @@ public partial class EventWaitHandle : WaitHandle
         public EventWaitHandle(bool initialState, EventResetMode mode, string name, out bool createdNew)
         {
             if (mode != EventResetMode.AutoReset && mode != EventResetMode.ManualReset)
-                throw new ArgumentException(SR.Argument_InvalidFlag, nameof(mode));
+                throw new ArgumentException(SR.Argument_InvalidFlag);
```

Comment:
Should we rather fix the test?

Reply:
Do we want the mode to be in the exception ?

Reply:
We typically include the argument name in the exception message.

Reply:
okay I will edit the test to include the ```mode``` parameter. But then we will have to make the change when the maestro brings the new version of coreclr into corefx